trade-finance-manager-ui/scripts/find-utilisation-reports-by-year-year-input-dropdown.ts

```diff
@@ -0,0 +1,29 @@
+const setupFindUtilisationReportsByYearYearInput = (): void => {
```

Comment:
`YearYear`?

Maybe could be called something like `addListAttributesToYearInputs`, `setupFindUtilisationReports` feels strange to me

Reply:
This script is specific only to the `find-utilisation-reports-by-year` page, and I think it's important to have the function and file name to reflect this. The function itself also does not actually add any attributes but instead sets up the event listeners, hence `setup`. The choice to include the fact that the input specifically being targeted by this script is the year input is that there are radio inputs for each of the banks on the page, so saying `input` by itself would not be clear

Reply:
Should we remove the dupicate `year-year` in the file and function name? It's a bit strange IMO.

I don't feel that naming reflects what it's doing.

The event listeners are called to add HTML attributes, to me the file name reads like it's just finding for something and nothing else.

Reply:
To match the other script which performs a similar action with respect to event listeners, I have renamed the file to `enable-find-utilisation-reports-by-year-dropdown-autocomplete` and function to `enableFindUtilisationReportsByYearDropdownAutocomplete`